README.md

```
@@ -99,6 +99,7 @@ and detailed information.
 
 ## Available Countries
 
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)